src/pudl/io_managers.py

```diff
@@ -414,6 +388,8 @@ def __init__(
                 exception, if the database is locked by another connection.  If another
                 connection opens a transaction to modify the database, it will be locked
                 until that transaction is committed.
+            settings: controls how parquet files are used by the PUDL ETL, in particular,
+                whether they should be used for input/output of dataframes.
```

Comment:
This looks like it might be a leftover from the previous PR that should be removed?